### DISCUSSION TITLE
error calling Concat: slice []interface {} not supported in concat

### DISCUSSION THREAD

**Discussion by coquinnsky** [google/docsy#1318] [Q&A]
When cloning this theme into our existing site, the js files in scripts.html can't be concatenated.
/hugo/themes/docsy/layouts/partials/scripts.html:68:107": execute of template failed: template: partials/scripts.html:68:107: executing "partials/scripts.html" at <resources.Concat>: error calling Concat: slice []interface {} not supported in concat

I've tried using module mounts as suggested in some other forums but can't seem to overcome this.

config 
---------
[module]

[[module.mounts]]
source = "static"
target = "static"

[[module.mounts]]
  source = "themes/docsy/assets/js"
  target = "assets/js"

line that causes issue
------------------------
{{ $js := (slice $jsBs $jsBase $jsAnchor $jsSearch $jsMermaid $jsPlantuml $jsMarkmap $jsDrawio) | resources.Concat "js/main.js" -}}

**Comment by dcrec1** [comment]
Im having the same problem with current version of Hugo (v0.114.0+extended) and Docsy master.

**Comment by DNF-SaS** [comment]
Did you find a solution for your problem? I'm encountering the same issue.

**Comment by coquinnsky** [comment]
Unfortunately I moved on from that project.  I've asked a previous colleague about it but he also doesn't remember, sorry.
